TeslaMateAgile/appsettings.json

```diff
@@ -21,5 +21,8 @@
   },
   "Barry": {
     "BaseUrl": "https://jsonrpc.barry.energy/json-rpc"
+  },
+  "Nordpool": {
+    "BaseUrl": "http://www.nordpoolspot.com/api/marketdata/page/10"
```

Comment:
What is the meaning of page 10 here, is it significant?

Reply:
To be honest ... I have no idea and no access to their API documentation. I have looked into a few npm and python implementations using the same URL for getting the hourly price so I assume it has something to do with the type of aggregation.

Reply:
If you look at the followed link, you will see in inspect mode, that the first page with alle geo areas is called page 10. 
https://www.nordpoolgroup.com/en/Market-data1/Dayahead/Area-Prices/ALL1/Hourly/?view=table

So its a way to fetch all areas at once. 

Besides that, I currently get 0 always in unit cost. So something is currently wrong. 



```
2022-11-24 22:14:02 | Calculated charge cost for 11/24/2022 16:00:00 UTC - 11/24/2022 17:00:00 UTC (unit cost: 0.00, fee per kWh: 1.10): 5.5094099316666664857900 for 5.00855448333333316890 energy
-- | --
  |   | 2022-11-24 22:14:02 | dbug: TeslaMateAgile.PriceHelper[0]
  |   | 2022-11-24 22:14:02 | Calculated charge cost for 11/24/2022 15:00:00 UTC - 11/24/2022 16:00:00 UTC (unit cost: 0.00, fee per kWh: 1.10): 3.0475272520000000202400 for 2.77047932000000001840 energy
  |   | 2022-11-24 22:14:02 | dbug: TeslaMateAgile.PriceHelper[0]
  |   | 2022-11-24 22:14:02 | Phase correction: 2 -> 3
  |   | 2022-11-24 22:14:02 | info: TeslaMateAgile.PriceHelper[0]
  |   | 2022-11-24 22:14:02 | 11/24/2022 16:00:00 UTC - 11/24/2022 17:00:00 UTC: 0.00
  |   | 2022-11-24 22:14:02 | dbug: TeslaMateAgile.PriceHelper[0]
  |   | 2022-11-24 22:14:02 | 11/24/2022 15:00:00 UTC - 11/24/2022 16:00:00 UTC: 0.00
  |   | 2022-11-24 22:14:02 | dbug: TeslaMateAgile.PriceHelper[0]
  |   | 2022-11-24 22:14:02 | Retrieved 2 prices:
  |   | 2022-11-24 22:14:02 | dbug: TeslaMateAgile.PriceHelper[0]
  |   | 2022-11-24 22:13:59 | Calculating cost for charges 11/24/2022 15:44:50 UTC - 11/24/2022 16:27:29 UTC
```